jwst/stpipe/tests/test_crds.py

```diff
@@ -34,6 +36,17 @@ def process(self, input_file):
         return datamodels.DataModel()
 
 
+def test_steppars_error_supression(_jail, caplog):
+    """Test supression of CRDS errors while retrieving step parameters"""
+    model = datamodels.ImageModel((10, 10))
+    model.save('image.fits')
```

Comment:
This needs to be done in a `with` context manager.  You're leaving an open file here.

Reply:
Will do, but technically no file is open, unless the below is a bug:

```
>>> import jwst.datamodels as dm
>>> m = dm.ImageModel((10, 10))
>>> m.save('junk.fits')
    'junk.fits'
>>> m._files_to_close
    []
```

Reply:
Interesting!  I was always under the impression that once something was written, it was then tied to the file.  This a nice, positive thing to discover!